ccm_web/client/src/components/ConfirmationTable.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useEffect, useState } from 'react'
+import { IconButton, makeStyles, Paper, Table, TableBody, TableCell, TableContainer, TableFooter, TableHead, TablePagination, TableRow, useTheme } from '@material-ui/core'
+import { KeyboardArrowRight, KeyboardArrowLeft, LastPage as LastPageIcon, FirstPage as FirstPageIcon } from '@material-ui/icons'
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexShrink: 0,
+    marginLeft: theme.spacing(2.5)
+  }
+}))
+
+interface StudentGrade {
+  rowNumber: number
+  uniqname: string
+  grade: string
+
+}
+
+interface ConfirmationTableProps {
+  grades: StudentGrade[]
+}
+
+interface TableHeaderColumnInfoShouldUseMatUIType {
+  id: keyof StudentGrade
+  label: string
+  minWidth: number
+  align?: 'left' | 'right' | undefined
+}
+
+const columns: TableHeaderColumnInfoShouldUseMatUIType[] = [
+  { id: 'rowNumber', label: 'Row Number', minWidth: 25 },
+  { id: 'uniqname', label: 'uniqname', minWidth: 100 },
+  { id: 'grade', label: 'grade', minWidth: 100 }
```

Comment:
I feel the Visually the header be styled(May be bigger font, bold etc) differently than row items, this is how it currently looks. @janelilr any Thoughts
![Screen Shot 2021-05-06 at 7 22 12 AM](https://user-images.githubusercontent.com/8579775/117290477-db457d80-ae3b-11eb-99a3-596de250cb32.png)

Reply:
I suggest the header be bold so that it pops out more. The font doesn't need to get bigger as long as the bolding is in place.

Reply:
In that case all the Header elements should be Bold and Column names should start with capital as well.